schemas/documents/shared/tag.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  name: 'tag',
+  type: 'document',
+  title: 'Tag',
+  fields: [
+    {
+      title: 'Name',
+      name: 'name',
+      type: 'string',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Color',
+      name: 'color',
+      type: 'string',
```

Comment:
there is a sanity plugin for picking colors: https://www.sanity.io/plugins/sanity-plugin-color-picker
probably more user friendly and less error prone that inputing the color manually, although that could be done in a follow-up issue, could you create one?
in the mean time, you can always add some description to this field to tell the user the formats that are accepted https://www.w3schools.com/cssref/css_colors_legal.asp. We could also validate the string using something like https://stackoverflow.com/a/56266358

Reply:
Sounds good. I left it as a string with validation, since we usually copy-paste a color picked by the designer. This worked best for the validation methods I tried: https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports

Reply:
new issue: regen-network/regen-web#762